source/_integrations/compensation.markdown

```diff
@@ -9,14 +9,40 @@ ha_release: '2021.5'
 ha_codeowners:
   - '@Petro31'
 ha_domain: compensation
+ha_config_flow: true
 ha_platforms:
   - sensor
-ha_integration_type: integration
+ha_integration_type: helper
 ---
 
 The **Compensation** {% term integration %} consumes the {% term state %} from other {% term sensors %}. It exports the compensated value as state in a separate {% term entity %} and the following values as attributes: `entity_id` and `coefficients`. A single polynomial, linear by default, is fit to all data points provided.
 
-## Configuration
+{% include integrations/config_flow.md %}
+
+Further information about these configuration options can be found under the [YAML configuration](#yaml-configuration) section.
+
+{% configuration_basic %}
+Name:
+  description: The name the sensor should have.
+Entity:
+  description: The entity that provides the input.
+Data points:
+  description: "The collection of data point conversions with the format `uncompensated_value, compensated_value`.  e.g., `1.0, 2.1`."
+Attribute:
+  description: Attribute from the source to monitor/compensate. When omitted, the state value of the source will be used.
+Degree:
+  description: "The degree of a polynomial. For example, Linear compensation (y = x + 3) has 1 degree, Quadratic compensation (y = x<sup>2</sup> + x + 3) has 2 degrees, etc."
+Precision:
+  description: Defines the precision of the calculated values, through the argument of round().
```

Comment:
Could you add an example for this one, maybe?

Reply:
I am rewriting it to make it less complicated so even without an example it should be understandable I think.